core/job/handler/v1beta1/job.go

```diff
@@ -60,7 +60,7 @@ type JobService interface {
 	GetByFilter(ctx context.Context, filters ...filter.FilterOpt) (jobSpecs []*job.Job, err error)
 	ReplaceAll(ctx context.Context, jobTenant tenant.Tenant, jobs []*job.Spec, jobNamesWithValidationError []job.Name, logWriter writer.LogWriter) error
 	Refresh(ctx context.Context, projectName tenant.ProjectName, namespaceNames, jobNames []string, logWriter writer.LogWriter) error
-	Validate(ctx context.Context, jobTenant tenant.Tenant, jobSpecs []*job.Spec, logWriter writer.LogWriter) error
+	Validate(ctx context.Context, jobTenant tenant.Tenant, jobSpecs []*job.Spec, jobNamesWithValidationError []job.Name, logWriter writer.LogWriter) error
```

Comment:
any reason why the name is `jobNamesWithValidationError`? since the job name we're passing haven't validated yet

Reply:
actually, there is a validation happening when we are trying to convert from proto to `job.Spec`. the name is actually make sense in `ReplaceAll` , but looks like not in `CheckJobSpecifications` context. how about I rename it to `jobNamesWithInvalidSpec`?

Reply:
sure, that kind of validation in proto conversion is different. Let's use different name, `jobNamesWithInvalidSpec` should be fine